### ISSUE TITLE
Getting started instructions fail on Mac OSX

### ISSUE THREAD

**Issue by JamieKnightBBC** [bbc/bbc-a11y#119]
## Summary

Getting Started Instructions don't work on Mac OSX El Capitan. 

## Expected Behavior

when the instructions say to run `bundle exec a11y` it should run without errors. 

## Current Behaviour

when the instructions say to run `bundle exec a11y` it fails with the error:

```
bundler: failed to load command: a11y (/usr/local/bin/a11y)
Cliver::Dependency::NotFound: Could not find an executable ["phantomjs"] on your path
```
## Possible Solution

I found phantom JS has not been installed in any recognised path. It has been installed in my home directory. I added: 

```
/Users/BBC/.phantomjs/2.1.1/darwin/bin:
```

To my path to ensure it was included and then the tests ran without issue.  

## Steps to Reproduce (for bugs)

1. Following install instructions on clean Mac OS X machine
2. Observe failures. 
## Your Environment

- Version used: Current as of August 2016
- Operating System and version: Mac OS El Capitan 10.11.5
- Link to your project: n/a

**Comment by joshski** [comment]
Thanks for reporting this @JamieKnightBBC 

You shouldn't need `phantomjs` in your path.

I think I've reproduced and [fixed](https://github.com/bbc/bbc-a11y/commit/90c65af1d163eceaae9efd9750b2555475e18648) this issue and I've pushed version 0.3.0 of the bbc-a11y gem with the fix.

Is there any chance you could try removing `phantomjs` from your path and see if the latest `bbc-a11y` still works without it?

**Comment by joshski** [comment]
I believe this is fixed, please reopen if you see this issue again
